packages/dds/tree/markdown-emitter.js

```diff
@@ -0,0 +1,227 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+const { MarkdownEmitter } = require("@fluid-tools/api-markdown-documenter");
+
+/**
+ * Custom {@link MarkdownEmitter} that generates HTML tables.
+ *
+ * @remarks Used by `./api-markdown-documenter.js`.
+ */
+class HugoMarkdownEmitter extends MarkdownEmitter {
```

Comment:
Note that a lot of what this particular emitter implementation is doing is specifically to tailor the docs output to be compatible with Hugo. I would probably start with just the standard `MarkdownEmitter`, and extend it if there are any specific formatting things you want to change / if there is special front-matter you want to append to the generated files.

Reply:
It's also doing a bunch of special handling so that we can generate HTML tables instead of basic markdown tables.